code/modules/clothing/masks/gasmask.dm

```diff
@@ -1,11 +1,11 @@
 /obj/item/clothing/mask/gas

```

Comment:
Why do we need to revert the sprite, when we can have a selection style system similar to the sechud/secgoggles where you can select your own sprite.